sys/include/net/hdlc/hdr.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2015 José Ignacio Alamos <jialaos@uc.cl>
+ * Copyright (C) 2018 Max van Kessel
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_hdlc_hdr    HDLC header
+ * @ingroup     net_hdlc
+ * @brief       HDLC header architecture
+ *
+ * @{
+ *
+ * @file
+ * @brief       Definitions for High Level Data Link Control header
+ *
+ * @author      José Ignacio Alamos <jialamos@uc.cl>
+ * @author      Max van Kessel
+ */
+#ifndef NET_HDLC_HDR_H
+#define NET_HDLC_HDR_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief HDLC types
+ */
+typedef enum {
+    HDLC_TYPE_RECEIVE_READY     = 0,        /**< ready receive type */
```

Comment:
I find the shifts quite confusing. Would you mind using hex values?

Reply:
I agree, this looks a bit weird here